lib/Core/Search/Elasticsearch/DocumentMapper/BlockFieldMapper/Aggregate.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Netgen\IbexaSearchExtra\Core\Search\Elasticsearch\DocumentMapper\BlockFieldMapper;
+
+use Ibexa\Contracts\Core\Persistence\Content as SPIContent;
+use Netgen\IbexaSearchExtra\Core\Search\Elasticsearch\DocumentMapper\BlockFieldMapper;
+
+class Aggregate extends BlockFieldMapper
+{
+    /**
+     * An array of aggregated field mappers, sorted by priority.
```

Comment:
These are not sorted by priority, I see that the same is documented in the Ibexa mappers, but it's worng, it should be removed.

Reply:
resolved in 6f13134c79d4f0d3d3a1738f7420faefc4f7f961